motioneye/sendtelegram.py

```diff
@@ -125,68 +116,50 @@ def on_media_files(media_files):
         and not snapshot_filename
         or snapshot_filename.startswith('%Y-%m-%d/')
     ):
-        moment = datetime.datetime.strptime(moment, '%Y-%m-%dT%H:%M:%S')
         prefix = moment.strftime('%Y-%m-%d')
         logging.debug('narrowing down still images path lookup to %s' % prefix)
 
-    ## mediafiles.list_media(camera_config, media_type='picture', prefix=prefix, callback=on_media_files)
     fut = utils.cast_future(
         mediafiles.list_media(camera_config, media_type='picture', prefix=prefix)
     )
     fut.add_done_callback(on_media_files)
-
     io_loop.start()
 
 
 def parse_options(parser, args):
+    parser.description='Send Telegram using bot api'
     parser.add_argument('api', help='telegram api key')
     parser.add_argument('chatid', help='telegram chat room id')
-    parser.add_argument('msg_id', help='the identifier of the message')
     parser.add_argument('motion_camera_id', help='the id of the motion camera')
-    parser.add_argument('moment', help='the moment in ISO-8601 format')
+    parser.add_argument('moment', help='the moment in ISO-8601 format', type=datetime.datetime.fromisoformat)
     parser.add_argument('timespan', help='picture collection time span')
     return parser.parse_args(args)
 
 
 def main(parser, args):
-    import meyectl
 
     # the motion daemon overrides SIGCHLD,
     # so we must restore it here,
     # or otherwise media listing won't work
     signal.signal(signal.SIGCHLD, signal.SIG_DFL)
 
-    if len(args) == 12:
-        # backwards compatibility with older configs lacking "from" field
-        _from = 'motionEye on {} <{}>'.format(
-            socket.gethostname(), args[7].split(',')[0]
-        )
-        args = args[:7] + [_from] + args[7:]
-
-    if not args[7]:
-        args[7] = 'motionEye on {} <{}>'.format(
-            socket.gethostname(), args[8].split(',')[0]
-        )
-
     options = parse_options(parser, args)
-    print(options)
+    
     meyectl.configure_logging('telegram', options.log_to_file)
-
     logging.debug('hello!')
```

Comment:
Looks like this debug log was nothing more than "reached here", becoming obsolete with the full options debug log below?
```suggestion
```